### PR TITLE
added admin

### LINKED ISSUE #14: Admin Schema and Route
### Description
Create admin schema, routes and controllers 

### Screenshots/testimonials
NA

### PR DESCRIPTION
**Pull request by ankiiitraj**
fixes #14 
Description: Added admin model, route and auth util

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [ ] I've tested my code on a Chromium based browser.
- [ ] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

### CODE REVIEW

**Review by prskid1000 — APPROVED** [review]
LGM

File: backend/util/auth.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = function authenticateJWT (req, res, next) {
```
Comment (EmperorYP7) [inline backend/util/auth.js:3]: This file is not being imported anywhere. 🤔

Comment (anktrj) [reply backend/util/auth.js:3]: This will be imported in place where auth is required such as deleting or creating notice, events etc

**Review by EmperorYP7 — APPROVED** [review]
LGTM!

**Pull request merged**